src/VivViewer.js

```diff
@@ -153,6 +164,10 @@ export default class VivViewer extends PureComponent {
         views={views}
         viewState={this.state.viewState}
       />
+    ) : (
+      <div className="viv-error">
+        <p>Viv does not work with Safari. Please use Chrome or Firefox.</p>
+      </div>
```

Comment:
- This might be gratuitous, but how would you feel about supplying this message via CSS content? That would let downstream users override, if they really needed it, without complicating our software interfaces.
- Maybe change the wording to blame Safari instead of blaming ourselves? "Safari does not support WebGL, which Viv requires. Chrome and Firefox are supported."

Reply:
Both of those are very good ideas.

Reply:
I think the first point is a bit more challenging.  I'm going to merge as-is in the interest of not sinking more time into this, but if you know of a good way to do this, let me know and/or open a PR.